### ISSUE TITLE
sataportcount error in virtualbox 5.0

### ISSUE THREAD

**Issue by kikitux** [mitchellh/packer#2429]
Virtualbox 5.0 is here, and builds in virtualbox fail with:

VBoxManage: error: Unknown option: --sataportcount

This was fixed at:
https://github.com/mitchellh/packer/commit/412d0d660775d6d10dd7b210b5b9c0d998f55b99

Seems the code is not ready for 5.0 version

```
	portCountArg := "--sataportcount"
	if strings.HasPrefix(version, "4.3") {
		portCountArg = "--portcount"
	}
```

https://github.com/mitchellh/packer/blob/edf3415c6e2248c8096f9c16cbad5022b73d6f21/builder/virtualbox/common/driver_4_2.go#L18-L37

**Comment by aarnaud** [comment]
same issue

**Comment by schiller256** [comment]
The same issue here after upgrade to VirtualBox 5

**Comment by gamename** [comment]
ditto. Same.

**Comment by tylert** [comment]
https://github.com/mitchellh/packer/pull/2438

**Comment by tylert** [comment]
+1... Downgrading to VirtualBox 4.3.30 in the meantime.

**Comment by tylert** [comment]
Say, @cbednarski, is there any chance of getting this pull request considered for 0.8.2?

**Comment by cbednarski** [comment]
@tylert Yeah we can include this.

**Comment by cbednarski** [comment]
This should be fixed by #2438, which is merged now.
